src/preprocess_data.py

```diff
@@ -48,10 +48,40 @@ def preprocess_data():
 
     raw_csv_files = glob.glob(f"{input_dir}/*.csv")
 
-    characters_data = pd.concat([pd.read_csv(file) for file in raw_csv_files],
-        ignore_index=True)
+    publisher_dfs = []
+    for csv_file in raw_csv_files:
+        if verbose: print(f"Processing {csv_file}")
+        publisher = os.path.basename(csv_file).split("-")[0]
+        publisher_df = pd.read_csv(csv_file)
+        publisher_df.columns = publisher_df.columns.str.lower().str.replace(' ', '_')
+        publisher_df = publisher_df.drop(["page_id",
+                                          "urlslug",
+                                          "id",
```

Comment:
I think ID might be handy to have, align can be related with public or secret identity. We may decide to drop those after EDA, within the pipe.

Reply:
Agreed, ID might be useful. How about keeping them for now?

Reply:
Okay, we'll keep it!